cst438_gradebook/src/main/resources/application.properties

```diff
@@ -5,3 +5,7 @@ server.port=8081
 spring.datasource.url=jdbc:h2:mem:testdb
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.show-sql=false
+
+# registration properties
+registration.url=http://localhost:8080/course
+registration.service=MQ
```

Comment:
Change to use REST instead of MQ so both services use the same communication services in the repository